backstage/community-plugins
# update sidebar filters for argocd plugin

This PR contains following changes

- Updated Resources filters in argocd plugin sidebar layout
- Add Name column to resources table
- Handled edge case in Sync status and Health status column, where value get undefined
- Added Unit tests and Refactor.

Screenshots:

![ScreenRecording2024-09-11at10 11 28PM-ezgif com-video-to-gif-converter](https://github.com/user-attachments/assets/edcbff5a-0b3c-43c1-9891-62625cd6d790)


How to test:

1. run yarn start:dev from workspaces/redhat-argocd
2. click on quarkus-app-prod application to see the sidebar changes.

## Hey, I just made a Pull Request!



#### :heavy_check_mark: Checklist
